### DISCUSSION TITLE
How to connect to PDB directly in thick mode?

### DISCUSSION THREAD

**Discussion by Stephenxman** [oracle/python-oracledb#93] [Q&A]
Thick mode, In version 18c+, I can connect directly to PDB when setting OS env ORACLE_PDB_SID=xxx, is there any way in 12c?
tks.

**Comment by cjbj** [comment]
Sorry I don't know.  
I did find https://mikedietrichde.com/2020/07/13/pitfalls-connect-to-a-pdb-directly-with-oracle_pdb_sid/ which has some caveats it looks useful to be aware of.
